_pages/coc.md

```diff
@@ -7,7 +7,82 @@ permalink: /coc/
 
 <div class="pretty-links">
 
-Coming soon! We're just polishing it up.
+Our Code of Conduct is our commitment to the community to provide a welcoming,
+safe space for all its participants. All speakers, sponsors and volunteers at
+StarCon are required to agree with the following code of conduct. Organizers
+will enforce this code throughout the event and social media spaces.
 
-</div>
+StarCon is dedicated to providing a harassment-free conference experience for
+everyone, regardless of gender, gender identity and expression, age, sexual
+orientation, disability, physical appearance, body size, race, ethnicity,
+religion (or lack thereof), or technology choices. We do not tolerate harassment
+of conference participants, speakers, or sponsors in any form. Sexual language
+and imagery is not appropriate for any conference venue, including talks,
+workshops, parties, and social media.
+
+## Unacceptable Behavior ##
+
+**Unacceptable behaviors include:** intimidating, harassing, abusive,
+discriminatory, derogatory or demeaning speech or actions by any participant in
+our community online, at all related events and in one-on-one communications
+carried out in the context of community business. Community event venues may be
+shared; please be respectful to all patrons of these locations.
+
+**Harassment includes:** harmful or prejudicial verbal or written comments
+related to gender, sexual orientation, race, religion, disability; inappropriate
+use of nudity and/or sexual images in public spaces (including presentation
+slides); deliberate intimidation, stalking or following; harassing photography
+or recording; sustained disruption of talks or other events; inappropriate
+physical contact, and unwelcome sexual attention.
+
+## Consequences of Unacceptable Behavior ##
+
+Unacceptable behavior from any community member, including sponsors and those
+with decision-making authority, will not be tolerated.
+
+Anyone asked to stop unacceptable behavior is expected to comply immediately.
+
+If a community member engages in unacceptable behavior, the community organizers
+may take any action they deem appropriate, up to and including a temporary ban
+or permanent expulsion from the conference without warning (and without refund
+in the case of a paid event).
+
+## Experiencing Unacceptable Behavior ##
+
+If you are subject to or witness unacceptable behavior, or have any other
+concerns, please [contact us](#contact-information).
+
+Additionally, community organizers are available to help community members
+engage with local law enforcement or to otherwise help those experiencing
+unacceptable behavior feel safe. In the context of in-person events, organizers
+will also provide escorts as desired by the person experiencing distress.
 
+## Addressing Grievances ##
+
+If you feel you have been falsely or unfairly accused of violating this Code of
+Conduct, [contact us](#contact-information) with a concise description of your
```

Comment:
I kinda like the explicit "if something bad happens you can email us"

without this section there's no "so what do I do"